### PR TITLE
Prevent player.setDateTime from setting invalid values

### PR DESCRIPTION
**Pull request by LeonMrBonnie**
See https://github.com/altmp/altv-issues/issues/364

### CODE REVIEW

**Comment by Lhoerion** [comment]
@LeonMrBonnie Add checks for other parameters too

**Review by Lhoerion — APPROVED** [review]
Personally I don't see a sense for including `algorithm` only for clamping, but I'll leave that to the devs

File: src/bindings/Player.cpp [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 #include "helpers/V8Helpers.h"
 #include "helpers/V8ResourceImpl.h"
+#include <algorithm>
```
Comment (Lhoerion) [inline src/bindings/Player.cpp:5]: Also I've seen that they tend to place bracket include above quoted ones

**Comment by 7Hazard** [comment]
> Personally I don't see a sense for including `algorithm` only for clamping, but I'll leave that to the devs

It's fine

**Pull request merged**